ezyfox-server-core/src/main/java/com/tvd12/ezyfoxserver/entity/EzySession.java

```diff
@@ -381,14 +380,6 @@ public interface EzySession extends
      */
     EzyConstant getDisconnectReason();
 
-    /**
-     * Get the lock of the session.
-     *
-     * @param name the lock name
-     * @return the lock
-     */
-    Lock getLock(String name);
```

Comment:
Just wanna double check if removing this method from interface would cause any backward compatibility issue?

Reply:
No, because currently there isn't any our projects is using it, I guess there is no one is using it because dev almost care about EzyUser not EzySession